chain-signatures/node/src/protocol/posit.rs

```diff
@@ -45,6 +45,7 @@ impl PositAction {
 pub enum PositInternalAction<S> {
     StartProtocol(Vec<Participant>, Positor<S>),
     Reply(PositAction),
+    Rejected,
```

Comment:
`Rejected` does not sound like an action. Sounds like a status of something.

Reply:
Yeah, I guess Abort would sound better. But if this is the only thing that needs changing, I'll change it in consequent PRs since I'd like this to go in before waiting on integrations to pass again